metecho/api/serializers.py

```diff
@@ -150,13 +154,65 @@ def get_sf_username(self, obj) -> Optional[str]:
             return None
         return obj.sf_username
 
+    def get_can_create_projects(self, obj) -> bool:
+        return obj.has_perm("api.add_project")
```

Comment:
@jerivas Is the plan for this to be manually set for every user in the admin? Or is this still WIP?

Reply:
David G. requested granular permissions in [earlier reviews](https://github.com/oddbird/Metecho/pull/1376#discussion_r687764034). The permission can be added directly to each user or as part of a group.

Reply:
I think that should work, but it means the Project would be created immediately in Metecho, and then provisioning would fail on GitHub, and then the Project would be deleted. By the later steps of the wizard the frontend knows which Orgs the user has access to, so they shouldn't even be able to hit that endpoint anyways

Reply:
Right, if we go this route I think we would either:

1. Check right away whether the user is a member of any of the GH orgs, and if not then don't show the "Create Project" button at all, OR...
2. Check when the modal first opens, since we need to know which Orgs to show in the dropdown list -- if the user is not a member of any of the GH orgs, then show a "sorry" message instead of allowing creation.

Reply:
@jgerigmeyer @jerivas I think it would make sense to check which orgs the user is a member of in the same rq job where we check which repositories they have access to (the one that is triggered when the user logs in or clicks the Re-Sync Projects button)

Reply:
Just pushed d70482d536ac5c3ae78b764bb07b1e57298f8716 that refreshes the user's orgs when the projects are refreshed.  The orgs will be sent as a separate WS message along the projects

Reply:
Also, we want to get rid of the Django permission now, right?

Reply:
> Do we want to keep the separate endpoint to refresh just orgs?

Sure, let's keep it for now. I think it may make sense to add a "Re-Sync GitHub Organizations" btn on the first page of the modal.